src/main/java/racing/domain/RacingCar.java

```diff
@@ -0,0 +1,35 @@
+package racing.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author han
+ */
+public class RacingCar {
+    private List<Car> cars;
+
+    private RacingCar(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    public static RacingCar of(List<Car> cars) {
+        return new RacingCar(cars);
+    }
+
+    public static RacingCar createByUsers(List<String> users) {
+        if (users == null || users.isEmpty()) {
+            return new RacingCar(Collections.emptyList());
+        }
+
+        List<Car> cars = users.stream()
+            .map(Car::create)
+            .collect(Collectors.toList());
+        return new RacingCar(cars);
```

Comment:
`RacingCar of(List<Car> cars)` 메서드가 주 생성자로 보이네요.
모든 생성자에서 new 키워드로 객체를 생성하기 보단 주 생성자를 통해 반환하는 것도 고민해 보시면 좋을 것 같아요.
```suggestion
        return of(cars);
```